yarn.lock

```diff
@@ -162,6 +162,20 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@apollo/experimental-nextjs-app-support@npm:0.4.1":
```

Comment:
Will Apollo have a package that isn't next "branded"?

Reply:
Yes eventually!